src/main.h

```diff
@@ -138,8 +144,12 @@ extern unsigned int nBytesPerSigOp;
 extern bool fCheckBlockIndex;
 extern bool fCheckpointsEnabled;
 extern size_t nCoinCacheUsage;
+/** A fee rate smaller than this is considered zero fee (for relaying, mining and transaction creation) */
 extern CFeeRate minRelayTxFee;
+/** Absolute maximum transaction fee (in satoshis) used by wallet and mempool (rejects high fee in sendrawtransaction) */
+extern CAmount maxTxFee;
 extern bool fAlerts;
+/* If the tip is older than this (in seconds), the node is considered to be in initial block download. */
```

Comment:
Good catch! Need to bookmark https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#doxygen-comments I guess.